x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/monitor_add_edit/form/field_config.tsx

```diff
@@ -1092,6 +1092,55 @@ export const FIELD = (readOnly?: boolean): FieldMap => ({
       },
     }),
   },
+  [ConfigKey.LABELS]: {
+    fieldKey: ConfigKey.LABELS,
+    label: i18n.translate('xpack.synthetics.monitorConfig.meta.label', {
+      defaultMessage: 'Label fields',
+    }),
+    controlled: true,
+    component: KeyValuePairsField,
+    helpText: i18n.translate('xpack.synthetics.monitorConfig.meta.helpText', {
+      defaultMessage:
+        'List of key-value pairs that will be sent with each monitor event. Useful for adding custom metadata to your monitor.',
+    }),
+    props: ({ field, setValue, trigger }): KeyValuePairsFieldProps => ({
+      readOnly,
+      keyLabel: i18n.translate('xpack.synthetics.monitorConfig.field.key.label', {
```

Comment:
nit: how about we use Key instead of Field as label itself is considered as field?